.travis.yml

```diff
@@ -29,7 +25,8 @@ addons:
       - libglib2.0-dev
 
 before_script:
-  - pip install flake8 ropper retdec-python keystone-engine capstone unicorn
+  - pip install flake8 ropper keystone-engine capstone unicorn
+  - if [ "$TRAVIS_PYTHON_VERSION" != "2.7" ]; then pip install retdec-python; fi
```

Comment:
Quick note, `retdec` is not necessary anymore since I moved it to [gef-extras](https://github.com/hugsy/gef-extras). 
I will also handle #231 to remove all references to `retdec` from GEF core. Sorry for the confusion